PAYNLSDK.IntegrationTests/V2/DirectDebit.cs

```diff
@@ -7,6 +7,8 @@ namespace PayNlSdk.IntegrationTests.V2;
 
 public class DirectDebit
 {
+	private const string MandateId = "IO-7149-8438-0752";
```

Comment:
Please replace this with an environment variable like in the TestHelper.cs, and do you mind updating the README with how to run these tests (what env vars to set, etc)